packages/devtools_shared/lib/src/memory/heap_sample.dart

```diff
@@ -31,7 +29,6 @@ class HeapSample {
 
   Map<String, dynamic> toJson() => <String, dynamic>{
         'timestamp': timestamp,
-        'rss': rss,
```

Comment:
rss should be stored to both the JSON files (that we export) and for reloading too.

Reply:
restored